client/src/app/site/motions/components/motion-change-recommendation/motion-change-recommendation.component.html

```diff
@@ -15,7 +15,7 @@ <h3>Change from line {{ lineRange.from }} to {{ lineRange.to }}:</h3>
     </form>
 </mat-dialog-content>
 <mat-dialog-actions>
-    <button mat-button mat-dialog-close>Abort</button>
+    <button mat-button mat-dialog-close translate>Abort</button>
```

Comment:
I would prefer "Cancel" over "Abort".

Reply:
I think @emanuelschuetze done this in the translation PR (#3989 )

Reply:
another pull request by @emanuelschuetze changed the label here.
Furthermore, `translation` somehow do not work on "button mat-button".
You need to put a <span> inside